FirebaseDatabase/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v8.4.0
+- [fixed] FirebaseDatabase `getData()` callbacks should be called on the main thread. (#8247)
```

Comment:
s/should be/are now/

Reply:
Also the 8.4.0 code freeze is past. 8.5.0 is targeted for 7/27